lib/order.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'menu'
+require_relative 'sms'
+
+class Order
+  attr_reader :basket
+
+  def initialize 
+    @basket = []
+    #eventually would be equal to a premade menu object (i.e Italian_restaurant) or would be able to select a premade menu
+    @menu = Menu.new   
```

Comment:
Ideally this would be dependency injected via an argument to `initialize` rather than hardcoded